jekyll/_cci2/server-3-operator-backup-and-restore.adoc

```diff
@@ -366,6 +371,121 @@ velero-68788b675c-dm2s7   1/1     Running   0          2m
 
 As restic is a daemonset, there should be one pod for each node in your Kubernetes cluster.
 
+////
+
+* S3-Compatible SETUP *
+
+////
+== Server 3.x backups with S3 Compatible Storage
+
+The following steps will assume you're using S3 compatible object storage, but
+no necessarily AWS S3, for your backups. We test and support https://min.io/[minio]
+for S3 compatible storage.
+
+These instructions were sourced from the Velero documentation https://velero.io/docs/v1.6/contributions/minio/[here].
+
+=== Step 1 - Configure `mc` client
+
+To start, configure `mc` to connect to your storage provider:
+
+[source,bash]
+----
+mc alias set my-provider <YOUR_ENDPOINT> <YOUR_ACCESS_KEY_ID> <YOUR_SECRET_ACCESS_KEY>
```

Comment:
nit: It might be useful to assign `my-provider` to a variable and use this instead for the rest of the examples

Reply:
Alright, tried to express that with a variable. Limme know what you think!

Reply:
Yup, that's better. You could have still called it `$PROVIDER` or something and also used vars in other places where parameters repeat themselves (e.g. `$BUCKET`), but we can definitely leave it like this :)